wpe/src/main/cpp/browser/entrypoints.cpp

```diff
@@ -195,20 +194,62 @@ void updateAllPageSettings(JNIEnv* env, jclass, jint pageId, jobject jPageSettin
 }
 } // namespace
 
-JNIEXPORT void JNICALL wpe_android_launchProcess(uint64_t pid, wpe::android::ProcessType processType, int* fd)
+struct AndroidProcessProvider {
+    struct wpe_process_provider *wpe_provider;
+};
+
+static void* process_provider_create(struct wpe_process_provider *process_provider)
```

Comment:
Better add the static function to the anonymous namespace above, which is the same as static in C++ but more common way of doing it.

And name it the same way as the other functions: `createProcessProvider()` (you've used the C-style naming with "namespace" prefixing).

Reply:
fixed